gr-utils/python/modtool/modtool_add.py

```diff
@@ -53,6 +53,8 @@ def setup_parser(self):
                 choices=self._block_types, default=None, help="One of %s." % ', '.join(self._block_types))
         ogroup.add_option("--license-file", type="string", default=None,
                 help="File containing the license header for every source code file.")
+        ogroup.add_option("--copyright", type="string", default=None,
```

Comment:
If `--license-file` is provided, then this option must be ignored. A quick check will help avoiding someone adding both arguments.